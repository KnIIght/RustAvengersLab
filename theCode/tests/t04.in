1
Thor
being the God of thunder
Mjolnir and Stormbreaker
