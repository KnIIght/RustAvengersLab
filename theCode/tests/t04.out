My name is Thor, my power is being the God of thunder and my weapons are Mjolnir and Stormbreaker.
You're big. I've fought bigger.
Thor is always worthy...
